donate.html

```diff
@@ -125,24 +131,18 @@
 
   <!-- Page Header -->
 
-  <div class="donate-header text-center">
+  <div class="hero">
     <div class="container">
-        <h2>
-          <span style="position: relative">
-          Donating to
-          <div class="d-none d-md-block">
-            <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
-            <div class="med-spark purp-spark" style="top: 50px; left: -120px;"></div>
-            <div class="small-spark blue-spark" style="top: 190px; left: -50px;"></div>
-            <div class="large-spark purp-spark" style="top: -35px; right: -40px;"></div>
-            <div class="small-spark pink-spark" style="top: 210px; right: -20px;"></div>
-            <div class="med-spark blue-spark" style="top: 90px; right: -110px;"></div>
-          </div>
-        </span>
-      </h2>
-        <img src="img/technica-logo.png" alt="Technica orb logo" class="top-logo" width="400" height="90">
-        <div class="b1"> <a target="_blank" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url" class="donate-button">Donate</a> </div>
+      <div class="hero-information">
+        <h1> Donating to </h1> -->
+        <div class="hero-logo">
+          <img id="technica-logo" src="img/technica-logo.png" alt="Technica logo" ALIGN = "top"> 
+        </div>
       </div>
+      <div class="hero-action text-center">
+        <button class="register-btn btn" onclick="location.href='https://register.gotechnica.org'">Donate</button>
```

Comment:
let's change the register-btn class to donate-btn and direct css to that class

Reply:
also looks like this is linking to the registration portal, replace with paypal link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url

Reply:
Oops yes I was gonna ask you about the actual link but totally forget